content/ja/docs/concepts/storage/persistent-volumes.md

```diff
@@ -376,9 +376,19 @@ PersistentVolumeは、リソースプロバイダーがサポートする方法
 
 アクセスモードは次の通りです。
 
-* ReadWriteOnce –ボリュームは単一のNodeで読み取り/書き込みとしてマウントできます
-* ReadOnlyMany –ボリュームは多数のNodeで読み取り専用としてマウントできます
-* ReadWriteMany –ボリュームは多数のNodeで読み取り/書き込みとしてマウントできます
+`ReadWriteOnce`
+: ボリュームは単一のNodeで読み取り/書き込みとしてマウントできます
+
+`ReadOnlyMany`
+: ボリュームは多数のNodeで読み取り専用としてマウントできます
+
+`ReadWriteMany`
+: ボリュームは多数のNodeで読み取り/書き込みとしてマウントできます
+
+`ReadWriteOncePod`
+: ボリュームは、単一のPodで読み取り/書き込みとしてマウントできます。クラスタ全体で1つのPodのみがそのPVCの読み取りまたは書き込みを行えるようにする場合は、ReadWriteOncePodアクセスモードを使用します。 これは、CSIボリュームとKubernetesバージョン1.22以降でのみサポートされます。
```

Comment:
```suggestion
: ボリュームは、単一のPodで読み取り/書き込みとしてマウントできます。クラスタ全体で1つのPodのみがそのPVCの読み取りまたは書き込みを行えるようにする場合は、ReadWriteOncePodアクセスモードを使用します。これは、CSIボリュームとKubernetesバージョン1.22以降でのみサポートされます。
```

Reply:
Thank you for your comments. I'll fix it.